archive/staging/22-6-30-deploy/config.json

```diff
@@ -1159,22 +1260,22 @@
           ]
         },
         "connections": [
-          "arbitrum-rinkeby",
-          "bsc-testnet",
+          "arbitrumrinkeby",
+          "bsctestnet",
           "evmostestnet",
           "goerli",
           "neontestnet",
-          "optimism-goerli",
+          "optimismgoerli",
           "rinkeby"
         ],
         "domain": 8881,
-        "name": "polygon-mumbai",
+        "name": "polygonmumbai",
         "specs": {
-          "blockExplorer": "https://polygon-mumbai.polygonscan.com/",
+          "blockExplorer": "https://polygonmumbai.polygonscan.com/",
```

Comment:
Neither of these URLs seem to work

Reply:
Ohh shoot will fix sorry, result of replace all

Reply:
Nvm not sure what the issue is there, if you could find correct ones and put in that'd be great